editoast/src/views/timetable/paced_train.rs

```diff
@@ -676,12 +675,23 @@ async fn project_path(
     Ok(Json(results))
 }
 
+/// Occupancy blocks output is described by blocks (signal updates)
+#[derive(Debug, Deserialize, Serialize, ToSchema)]
+pub struct OccupancyBlocksPacedTrainResult {
+    /// Paced train
+    #[schema(value_type = Vec<SignalUpdate>)]
+    pub paced_train: OccupancyBlocks,
+    /// Exceptions whose blocks are different from the paced train
+    #[schema(value_type = HashMap<String, Vec<SignalUpdate>>)]
+    pub exceptions: HashMap<String, OccupancyBlocks>,
+}
+
 #[utoipa::path(
     post, path = "",
     tag = "paced_train",
     request_body = OccupancyBlockForm,
     responses(
-        (status = 200, body = HashMap<i64, OccupancyBlocks>),
+        (status = 200, body = HashMap<i64, OccupancyBlocksPacedTrainResult>),
     ),
 )]
 async fn occupancy_blocks(
```

Comment:
Do we have some tests that go through the evolution of this endpoint. Because I'm surprised to not see tests changing in the PR.

Reply:
Super, thank you!